js/utils.js

```diff
@@ -48,11 +48,28 @@ export function setattr(obj, name, value) {
     obj[name] = value;
 }
 
+/**
+ * Parse a query string into an object
+ * @param  {String} qs  A querystring, with or without the leading '?'
+ * @return {Object}     The parsed query string
+ */
+export function parseQS(qs) {
```

Comment:
Indeed, it's definitely bigger https://github.com/WebReflection/url-search-params

I wonder if there are things to pick in it though.